hack/label-workers.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+script_dir=$(dirname "$(readlink -f "$0")")
+kubectl=$script_dir/../kubevirtci/cluster-up/kubectl.sh
+
+$kubectl label node -l '!node-role.kubernetes.io/master' node-role.kubernetes.io/worker=''
```

Comment:
What about single-node setup? Tests should still work on local env where 1 node is the default.

If there is only single node, let's make it both master and worker?

Reply:
Totally, also for the k8s-1.17 provider (we are not going to use this provider since it's centos8 node based).